docs/DIRECTION.md

```
@@ -0,0 +1,119 @@
+# Direction — Where GPT-Trader Is Going
+
+---
+status: current
```

markdownlint-cli2 (0.22.1)

[warning] Heading style
Expected: atx; Actual: setext

(MD003, heading-style)